p2p/src/main/java/bisq/network/p2p/storage/P2PDataStorage.java

```diff
@@ -580,6 +577,15 @@ private boolean addProtectedStorageEntry(ProtectedStorageEntry protectedStorageE
             return false;
         }
 
+        // To avoid that expired data get stored and broadcast we check early for expire date.
+        if (protectedStorageEntry.isExpired(clock)) {
+            log.warn("We received an expired protectedStorageEntry from ppper {}. getCreationTimeStamp={}, protectedStorageEntry={}",
```

Comment:
I guess you meant peer.
```suggestion
            log.warn("We received an expired protectedStorageEntry from peer {}. getCreationTimeStamp={}, protectedStorageEntry={}",
```